### PR TITLE
NODE-623: Network specific config defaults

### PR DESCRIPTION
**Pull request by Karasiq**

### CODE REVIEW

File: node/src/main/scala/com/wavesplatform/settings/package.scala [hunk 1]
```diff
@@ -44,14 +44,20 @@ package object settings {
...
 
   def loadConfig(maybeUserConfig: Option[Config]): Config = {
-    val directoryDefaults = ConfigFactory
-      .parseString(s"waves.directory = $defaultDirectory")
+    val sysProps = ConfigFactory.defaultOverrides()
 
-    val defaults = ConfigFactory.defaultOverrides()
+    val external = maybeUserConfig
+      .fold(sysProps)(sysProps.withFallback)
+      .withFallback(ConfigFactory.parseString(s"waves.directory = $defaultDirectory"))
```
Comment (Karasiq) [inline node/src/main/scala/com/wavesplatform/settings/package.scala]: Need to merge with the network-dependent default directory fix

File: node/src/main/resources/network-defaults.conf [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+      # Network address to bind to
+      bind-address = "127.0.0.1"
+
+      # Port to listen to REST API requests
+      port = 6869
+    }
+
+    features {
+      supported = [2] # NG
```
Comment (DrBlast) [inline node/src/main/resources/network-defaults.conf:186]: remove activated feature

File: node/src/main/resources/application.conf [hunk 3]
```diff
@@ -129,7 +133,7 @@ waves {
     file = ${waves.directory}"/wallet/wallet.dat"
 
     # Password to protect wallet file
-    # password = "some string as password"
+    password = "ridethewaves!"
```
Comment (DrBlast) [inline node/src/main/resources/application.conf:136]: remove password

File: node/src/main/resources/network-defaults.conf [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+            1 = 0
+            2 = 0
+            3 = 0
+            4 = 0
+            5 = 0
+            6 = 0
+            7 = 0
+            8 = 0
+            9 = 0
+            10 = 0
```
Comment (DrBlast) [inline node/src/main/resources/network-defaults.conf:56]: add preactivated features 11-15

File: node/src/main/resources/network-defaults.conf [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+          }
+          double-features-periods-after-height = 1000000000
+          max-transaction-time-back-offset = 120m
+          max-transaction-time-forward-offset = 90m
+        }
```
Comment (DrBlast) [inline node/src/main/resources/network-defaults.conf:61]: add rewards section for devnet config. Compare network configs with master before move to other file

File: node/src/main/resources/network-defaults.conf [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+  devnet {
+    # Blockchain settings
+    blockchain {
+      type = CUSTOM
+      custom {
+        address-scheme-character = "D"
+        functionality {
+          feature-check-blocks-period = 5
+          blocks-for-feature-activation = 4
+          generation-balance-depth-from-50-to-1000-after-height = 0
```
Comment (DrBlast) [inline node/src/main/resources/network-defaults.conf:24]: add  `min-asset-info-update-interval = 7` to devnet.custom.functionality

**Pull request merged**